test/run_envoy_tests.sh

```diff
@@ -61,10 +61,16 @@ then
   exit 0
 fi
 
+# TODO(htuch): Clean this up when Bazelifying the hot restart test below.
+HOT_RESTART_JSON="$TEST_SRCDIR"/test/config/integration/hot_restart.json
+cat "$TEST_TMPDIR"/test/config/integration/server.json |
```

Comment:
TODO added. One idea is we could add a flag to allow us to write back to the filesystem (or print to stdout) the JSON corresponding to what's actually used when zero port binding - basically the same JSON object we parsed but with addresses replaced based on the discovered ports. This would generalize to be useful in other cross-process integration tests where we can't access the `ListenSocket` objects directly. Another alternative is to do log scraping and JSON templating outside of Envoy. What do you think?